src/util.c

```diff
@@ -1159,7 +1160,15 @@ x_realpath(const char *path)
 	p = realpath(path, ret);
 #elif defined(_WIN32)
 	if (path[0] == '/') {
-		path++;  // Skip leading slash.
+		char *p = NULL;
```

Comment:
This looks like a fix that is unrelated to the issue of `-MF`? Please create a separate pull request for this instead.